map.json

```diff
@@ -0,0 +1,117 @@
+[
+    {
+        "version": 2,
+        "path": "nvram3.bin",
+        "entries": [
+            {
+                "name": "_version",
+                "offset": 53,
+                "size": 1
+            },
+            {
+                "name": "save3_bool",
+                "offset": 54,
+                "size": 1,
+                "bit_offset": 0,
+                "is_bool": true
```

Comment:
I've put it just to avoiding define `bit_size: 1` and checking if `bit_size` is really `1`. Maybe another name, like `bit_field`? I'm open to suggestions. That said, maybe would be interesting have a `type` property, which would define `size` automatically, addressing @lpereira concern. I'm not sure about it because user, right now, has to know size to calc offsets.